CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ All notable changes to this project will be documented in this file.
 ### Fixed
 
 ### Changed
-
+-   #1355 : remove `BasicParser` arguments.
```

Comment:
See previous comment. This should be capitalised really. I also think it is important to mention that these are unused arguments

Reply:
ok